### PR TITLE
[DO NOT MERGE] Propose new sample format.

### PR DESCRIPTION
**Pull request by jmdobry**
_Some_ of the notes for the reasoning behind this proposal are in https://github.com/googleapis/nodejs-dlp/pull/154.

More detailed notes can be provided upon request.

This PR demonstrates a new sample format for LRO-based samples.

### CODE REVIEW

File: samples/speech_transcribe_async.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// Imports the Google Cloud Speech API client library
+const speech = require('@google-cloud/speech');
+
+// Import other required libraries
+const fs = require('fs');
+
+/**
+ *  Inspects the provided text.
+ *
+ * @param filepath {string} The path to the file to inspect.
```
Comment (beccasaurus) [inline samples/speech_transcribe_async.js]: ✂️ 

```js
/**
 * Transcribing Long Audio File
 *
 * @param filepath {string} The path to audio file to transcribe.
 * @param encoding {string} The AudioEncoding to use, e.g. FLAC.
 * @param sampleRateHertz {string} Sample rate in Hertz of the audio data sent.
 * @param languageCode {string} The language of the supplied audio, e.g. 'en-US'
 * @returns {object} The LongRunningRecognizeResponse.
*/
async function main(...)
```

File: samples/speech_transcribe_async.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      content: fileBytes,
+    },
+    config: {
+      encoding: encoding,
+      sampleRateHertz: sampleRateHertz,
+      languageCode: languageCode,
+    },
+  };
+
+  // Detects speech in the audio file. This creates a recognition job that you
```
Comment (beccasaurus) [inline samples/speech_transcribe_async.js]: > Detects speech in the audio file. This creates a recognition job that you can wait for now, or get its result later.

Can we use a generic comment for LRO operations?

e.g.

```js
// Wait for operation to complete.
const [operation] = await client.longRunningRecognize(request);
```

Comment (beccasaurus) [reply samples/speech_transcribe_async.js]: Similar to how getting the response from the operation below has a nice, generic comment: `// Get a Promise representation of the final result of the job`

**Comment by beccasaurus** [comment]
Meta Comment :: there's some inconsistency in comments that do/don't end in a `.`

Authoring a comment, I'd love to not have to think about whether or not I should use a `.`

Maybe let's always end comments in a `.` to keep it simple?

File: samples/speech_transcribe_async.js (hunk 2 above)
Comment (jmdobry) [reply samples/speech_transcribe_async.js]: Done, I think. Take a look.

File: samples/speech_transcribe_async.js (hunk 1 above)
Comment (jmdobry) [reply samples/speech_transcribe_async.js]: Done

File: samples/speech_transcribe_async.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+// [START speech_transcribe_async]
+// Imports the Google Cloud Speech API client library
+const speech = require('@google-cloud/speech');
+
+// Import other required libraries
+const fs = require('fs');
+
+/**
+ *  Inspects the provided text.
```
[the remark this answers is not in the document]
Comment (jmdobry) [reply samples/speech_transcribe_async.js]: Done

**Comment by JustinBeckwith** [comment]
This has been hanging around for a loooong time.  We'll keep the branch, and feel free to re-open the PR when it's ready to go.

**Comment by jmdobry** [comment]
Branch has been updated with latest sample guidance. Re-opening for a final review in concert with googleapis/nodejs-dlp#154

**Pull request closed without merging**